src/App.js

```diff
@@ -1,23 +1,49 @@
-import logo from './logo.svg';
-import './App.css';
+import "./App.css";
+import React, { useState } from "react";
+import contacts from "./contacts.json";
+import Contacts from "./components/Contacts";
 
 function App() {
+  const [contact, setContact] = useState(contacts.slice(0, 5));
+  const createNew = () => {
+    const newIndex = Math.floor(Math.random() * contacts.length);
```

Comment:
A good idea here is to apply Math.random only on the remaining contacts on the list, this way you can get repeated contacts